### ISSUE TITLE
Getting SyntaxWarning: invalid escape sequence '\ '

### ISSUE THREAD

**Issue by thesayyn** [bazelbuild/rules_pkg#829]
After upgrading python to `3.12.2` i started getting these annoying warning messages; 

```
INFO: From Writing: bazel-out/darwin_arm64-opt-exec-2B5CBBC6/bin/external/jetty/jetty.tar [for tool]:
/private/var/tmp/_bazel_thesayyn/9000763e5f5851865f9d806cb10059e8/sandbox/darwin-sandbox/2378/execroot/distroless/bazel-out/darwin_arm64-opt-exec-2B5CBBC6/bin/external/rules_pkg/pkg/private/tar/build_tar.runfiles/rules_pkg/pkg/private/tar/build_tar.py:29: SyntaxWarning: invalid escape sequence '\ '
  """Normalize a path to the format we need it.

```

**Comment by aiuto** [comment]
I think this was fixed by #815.   Have you tried the latest release or from head?

**Comment by AFMiziara** [comment]
Using the latest version fixed the warning message for me.
